aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -89,9 +91,8 @@ impl VersionedValue {
         match self.versioned_map.entry(txn_idx) {
             Entry::Occupied(mut o) => {
                 let bypass = match &**o.get() {
-                    Value(_, maybe_delta) => maybe_delta.map_or(NoBypass, DeltaBypass),
-                    Delta(delta) => DeltaBypass(*delta),
-                    Snapshot(id, delta) => SnapshotBypass(*id, *delta),
+                    Value(_, maybe_apply) => maybe_apply.clone().map_or(NoBypass, Bypass),
```

Comment:
What does this statement mean?

Reply:
map_or(default, func)

so it takes maybe_apply, if it is one, returns NoBypass, if it is |a| returns Bypass(a)

basically 

                    Value(_, maybe_apply) => maybe_apply.clone().map_or(NoBypass, |v| Bypass(v)),